tests/conftest.py

```diff
@@ -15,6 +16,13 @@ def admin_client() -> DynamicClient:
     return get_client()
 
 
+@pytest.fixture(scope="session")
```

Comment:
>you can check by getting "oc whoami" first

Well, that'd only tell us the name of the user, not if it was an admin or not.

Reply:
With this command we could get the list of cluster admin users
`oc get clusterrolebindings -o json |jq '.items[] |select(.roleRef.name=="cluster-admin") |.subjects[] |select(.kind=="User") |.name'`
but honestly it makes more sense to me to rename this to `current_client_token`

Reply:
in https://github.com/opendatahub-io/opendatahub-tests/pull/54/files#r1881639137 i meant to check if the current context is admin then return the token.

so if you do not need admin, but current token, please use https://github.com/opendatahub-io/opendatahub-tests/blob/main/tests/trustyai/conftest.py#L55

Reply:
I don't agree in using a fixture from the trustyai folder for an operation that everyone will need sooner or later. I think in future PRs it would be better to refactor TrustyAI to use this new fixture.